src/Psalm/Internal/Analyzer/Statements/Expression/Call/Method/MethodCallReturnTypeFetcher.php

```diff
@@ -92,6 +95,16 @@ public static function fetch(
             }
         }
 
+        if ($premixin_method_id->method_name === 'getcode'
+            && $premixin_method_id->fq_class_name !== Exception::class
+            && in_array(Throwable::class, $class_storage->class_implements)) {
+            if ($premixin_method_id->fq_class_name === PDOException::class) {
```

Comment:
Please use string so psalm doesn't crash if the PDO extension isn't loaded.

Reply:
`::class` doesn't call the autoloader so it shouldn't be an issue